src/sentry/models/useroption.py

```diff
@@ -21,6 +21,24 @@ class UserOptionValue(object):
     # 'workflow:notifications'
     all_conversations = '0'
     participating_only = '1'
+    # 'deploy-emails
+    all_deploys = '2'
+    committed_deploys_only = '3'
+    no_deploys = '4'
+
+option_scope_error = 'this is not a supported use case, scope to project OR organization'
+
+
+def user_metakey(user):
+    return (user.pk)
```

Comment:
the same function is being used for getting and setting from the dict. i'm no python god but 
 think that a primitive value should work just as well as a tuple here